invenio/base/config.py

```diff
@@ -63,6 +63,7 @@
 PACKAGES = [
     'invenio_records',
     'invenio_comments',
+    'invenio_pages',
```

Comment:
We should not force people to use this package as it was disabled by default. We can maybe add it to `extras_require = {..., 'pages': ['invenio-pages>=0.1.0'], ...}`.